### PR TITLE
update readme

### PR DESCRIPTION
**Pull request by johnbampton**
added myself (Pedram) to the list!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -60,6 +60,12 @@ It might be worth mentioning that I use software from **Serif Software** in the
 
 You can find me at the PC periodically throughout the day and night, about six days a week, right here in Arizona.
 
+### Adorner, Electrical Engineer, Self Pproclaimed Utility Wrench,  Python Newbie, MATLAB Ninja, IT guy
+- Pedram-cVx (https://www.codewars.com/users/pedramcvx1)
```
Comment (jbampton) [inline README.md]:
```suggestion
- [Pedram-cVx](https://www.codewars.com/users/pedramcvx1)
```

File: README.md [hunk 2]
```diff
@@ -60,6 +60,12 @@ It might be worth mentioning that I use software from **Serif Software** in the
 
 You can find me at the PC periodically throughout the day and night, about six days a week, right here in Arizona.
 
+### Adorner, Electrical Engineer, Self Pproclaimed Utility Wrench,  Python Newbie, MATLAB Ninja, IT guy
```
Comment (jbampton) [inline README.md]:
```suggestion
### Adorner, Electrical Engineer, Self Pproclaimed Utility Wrench, Python Newbie, MATLAB Ninja, IT guy

```

File: README.md [hunk 3]
```diff
@@ -60,6 +60,12 @@ It might be worth mentioning that I use software from **Serif Software** in the
...
+
```
Comment (jbampton) [inline README.md:66]:
```suggestion
```

**Review by MohammadLotfiA — APPROVED** [review]
LGTM!

File: README.md (hunk 3 above)
Comment (MohammadLotfiA) [reply README.md:66]: got it

**Review by MohammadLotfiA — APPROVED** [review]
got it

File: README.md [hunk 4]
```diff
@@ -60,6 +60,12 @@ It might be worth mentioning that I use software from **Serif Software** in the
 
 You can find me at the PC periodically throughout the day and night, about six days a week, right here in Arizona.
 
+### Adorner, Electrical Engineer, Self Pproclaimed Utility Wrench, Python Newbie, MATLAB Ninja, IT guy
```
Comment (jbampton) [inline README.md]:
```suggestion
### Adorner, Electrical Engineer, Self Proclaimed Utility Wrench, Python Newbie, MATLAB Ninja, IT guy
```

File: README.md [hunk 5]
```diff
@@ -66,7 +66,6 @@ You can find me at the PC periodically throughout the day and night, about six d
...
 
-
```
Comment (MohammadLotfiA) [inline README.md]: do you want to remove this?

**Review by MohammadLotfiA — DISMISSED** [review]

**Review by MohammadLotfiA — APPROVED** [review]

**Review by jbampton — APPROVED** [review]
🌊

**Comment by MohammadLotfiA** [comment]
I gotta be more careful when writing. these typos are embarrassing!!

**Comment by jbampton** [comment]
I will add two spell checkers and `pre-commit`

**Pull request merged**